### ISSUE TITLE
[B60][stock PyTorch][XPU] Inductor worker crashes in conv2d/opinfo/strided-block tests

### ISSUE THREAD

**Issue by mengfei25** [intel/torch-xpu-ops#4793]
## Summary

B60 shows multiple stock PyTorch XPU Inductor tests where pytest workers crash during setup. The same tests passed on PVC.

## Failure Category

Worker process crash during test setup.

Representative message:

```text
failed on setup with "worker 'gwN' crashed while running ..."
```

## Scope

- Test suite: stock PyTorch
- Platform comparison: B60 error; PVC passed
- Affected rows observed: 9
- Affected areas:
  - `test/inductor/test_torchinductor_opinfo.py`
  - `test/inductor/test_torchinductor.py`
  - `test/inductor/test_torchinductor_strided_blocks.py`

## Representative Failures

```text
test/inductor/test_torchinductor_opinfo.py::TestInductorOpInfoXPU::test_comprehensive_nn_functional_max_pool3d_xpu_float64
failed on setup with "worker 'gw12' crashed while running ..."

test/inductor/test_torchinductor.py::GPUTests::test_conv2d_backward_parametrized_dilation_1_stride_2_padding_0_kernel_3_channels_groups4_nhwc_True_xpu
failed on setup with "worker 'gw16' crashed while running ..."

test/inductor/test_torchinductor_strided_blocks.py::TritonBlockPointerTestGPU::test_2d_reduction_multi_kernel_xpu
failed on setup with "worker 'gw9' crashed while running ..."
```

## Expected Behavior

The tests should complete normally or fail with a Python-level assertion/error. The pytest worker process should not crash.

## Notes

Most crash rows are conv2d backward parametrizations with grouped convolution, NHWC layout, and varied dilation/stride/padding. This may point to a common generated-kernel or runtime crash path.

## Full Case List

Total cases: 9

1. `stock pytorch` | `test/inductor/test_torchinductor_opinfo.py::TestInductorOpInfoXPU::test_comprehensive_nn_functional_max_pool3d_xpu_float64` | `error`
   - `failed on setup with "worker 'gw12' crashed while running 'test/inductor/test_torchinductor_opinfo.py::TestInductorOpInfoXPU::test_comprehensive_nn_functional_max_pool3d_xpu_float64'"`
2. `stock pytorch` | `test/inductor/test_torchinductor.py::GPUTests::test_conv2d_backward_parametrized_dilation_1_stride_2_padding_0_kernel_3_channels_groups4_nhwc_True_xpu` | `error`
   - `failed on setup with "worker 'gw16' crashed while running 'test/inductor/test_torchinductor.py::GPUTests::test_conv2d_backward_parametrized_dilation_1_stride_2_padding_0_kernel_3_channels_groups4_nhwc`
3. `stock pytorch` | `test/inductor/test_torchinductor.py::GPUTests::test_conv2d_backward_parametrized_dilation_2_stride_2_padding_1_kernel_3_channels_groups4_nhwc_True_xpu` | `error`
   - `failed on setup with "worker 'gw17' crashed while running 'test/inductor/test_torchinductor.py::GPUTests::test_conv2d_backward_parametrized_dilation_2_stride_2_padding_1_kernel_3_channels_groups4_nhwc`
4. `stock pytorch` | `test/inductor/test_torchinductor.py::GPUTests::test_conv2d_backward_parametrized_dilation_1_stride_1_padding_0_kernel_3_channels_groups4_nhwc_True_xpu` | `error`
   - `failed on setup with "worker 'gw18' crashed while running 'test/inductor/test_torchinductor.py::GPUTests::test_conv2d_backward_parametrized_dilation_1_stride_1_padding_0_kernel_3_channels_groups4_nhwc`
5. `stock pytorch` | `test/inductor/test_torchinductor.py::GPUTests::test_conv2d_backward_parametrized_dilation_2_stride_1_padding_0_kernel_3_channels_groups4_nhwc_True_xpu` | `error`
   - `failed on setup with "worker 'gw19' crashed while running 'test/inductor/test_torchinductor.py::GPUTests::test_conv2d_backward_parametrized_dilation_2_stride_1_padding_0_kernel_3_channels_groups4_nhwc`
6. `stock pytorch` | `test/inductor/test_torchinductor.py::GPUTests::test_conv2d_backward_parametrized_dilation_1_stride_2_padding_1_kernel_3_channels_groups4_nhwc_True_xpu` | `error`
   - `failed on setup with "worker 'gw20' crashed while running 'test/inductor/test_torchinductor.py::GPUTests::test_conv2d_backward_parametrized_dilation_1_stride_2_padding_1_kernel_3_channels_groups4_nhwc`
7. `stock pytorch` | `test/inductor/test_torchinductor.py::GPUTests::test_conv2d_backward_parametrized_dilation_2_stride_2_padding_0_kernel_3_channels_groups4_nhwc_True_xpu` | `error`
   - `failed on setup with "worker 'gw21' crashed while running 'test/inductor/test_torchinductor.py::GPUTests::test_conv2d_backward_parametrized_dilation_2_stride_2_padding_0_kernel_3_channels_groups4_nhwc`
8. `stock pytorch` | `test/inductor/test_torchinductor.py::GPUTests::test_conv2d_backward_parametrized_dilation_1_stride_1_padding_1_kernel_3_channels_groups4_nhwc_True_xpu` | `error`
   - `failed on setup with "worker 'gw22' crashed while running 'test/inductor/test_torchinductor.py::GPUTests::test_conv2d_backward_parametrized_dilation_1_stride_1_padding_1_kernel_3_channels_groups4_nhwc`
9. `stock pytorch` | `test/inductor/test_torchinductor_strided_blocks.py::TritonBlockPointerTestGPU::test_2d_reduction_multi_kernel_xpu` | `error`
   - `failed on setup with "worker 'gw9' crashed while running 'test/inductor/test_torchinductor_strided_blocks.py::TritonBlockPointerTestGPU::test_2d_reduction_multi_kernel_xpu'"`

## Reproducer / Environment Setup

```bash
docker run -it -e TZ=Asia/Shanghai --device=/dev/mem --device=/dev/dri --group-add video --privileged -v $(realpath ${HOME}):/home/jenkins --shm-size=8g intelgpu/ubuntu-26.04-rolling:26.18 bash

apt update
apt upgrade -y

curl -LsSf https://astral.sh/uv/install.sh | sh
source $HOME/.local/bin/env
uv venv myvenv --python 3.10 --clear
source myvenv/bin/activate
uv pip install pip wheel "setuptools<81"
uv pip install torch==2.13.0 torchaudio torchvision --index-url https://download.pytorch.org/whl/xpu

pytorch_commit="$(python -c 'import torch; print(torch.version.git_version)')"
git clone https://github.com/pytorch/pytorch
cd pytorch
git checkout ${pytorch_commit}

uv pip install -r .ci/docker/requirements-ci.txt
uv pip install -U typing_extensions
uv pip install pytest pytest-timeout pytest-xdist pytest-rerunfailures

python test/inductor/test_torchinductor_opinfo.py -k 'test_comprehensive_nn_functional_max_pool3d_xpu_float64'
python test/inductor/test_torchinductor.py -k 'conv2d_backward_parametrized and groups4 and nhwc_True_xpu'
python test/inductor/test_torchinductor_strided_blocks.py -k 'test_2d_reduction_multi_kernel_xpu'
```
